enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -188,6 +188,9 @@ private void applyQueuedTransactions() throws IOException
             // Check rotation explicitly, since the version of append that we're calling isn't doing that.
             logRotation.rotateLogIfNeeded( LogAppendEvent.NULL );
 
+            // Check kernel health after log rotation
+            kernelHealth.assertHealthy( IOException.class );
```

Comment:
@davidegrohmann  @tinwelint i was looking into `panicMessage` and i would not treat that message as 'instance dead' or restart or we warn you! It's some problem, has something to do with  a tx.
As part of this pr we can just update that message to be more scary-realistic-descriptive, but in general i believe we need to provide more accurate useful specific messages at least and behaves in different way depending from exception encountered.